app/src/api/dataSource/ClientApiDataSource.ts

```diff
@@ -43,7 +42,7 @@ export class ClientApiDataSource implements ClientApi {
 
     const response = await getJsonRpcClient().query<FeedRequest, Post[]>(
       {
-        contextId,
+        contextId: getContextId(),
```

Comment:
Like this its more code but refetched on every request and preventing possible errors